index.js

```diff
@@ -1,27 +1,33 @@
-var fs = require('fs')
-var ncp = require('ncp')
+var fs = require('fs-extra')
 var path = require('path')
 var mkdirp = require('mkdirp')
 
 module.exports = {}
 
+module.exports.FALLBACK_DIR_NAME = 'MAPEO_PRESETS_FALLBACK_DIR'
+
 module.exports.unpack = function (root, cb) {
-  var presetRoot = path.join(root, 'presets')
+  // We need this because mapeo-default-settings does not have
+  // a subdirectory, but styles in this repo does have a subdirectory for
+  // each default style.
+  var presetRoot = path.join(root, 'presets', module.exports.FALLBACK_DIR_NAME)
```

Comment:
This also ensures we don't override someone's files as this is always an autogenerated directory